src/network/MissionControlProtocol.h

```diff
@@ -56,7 +59,7 @@ class MissionControlProtocol : public WebSocketProtocol { // TODO: add documenta
 	void handleCameraStreamCloseRequest(const json& j);
 	void handleJointPowerRequest(const json& j);
 	void handleDriveRequest(const json& j);
-	void sendCameraStreamReport(const CameraID& cam, const std::string& b64_data);
+	void sendCameraStreamReport(const CameraID& cam, const std::basic_string<uint8_t>& nal_data);
```

Comment:
Can we use a slightly more descriptive name than `nal_data`? How about `frameData` or something? Anyone who hasn't worked with video encoders won't know what `nal` means.